__tests__/testCases/completeDecimalInput.ts

```diff
@@ -36,43 +36,25 @@ export const completeDecimalInput: TaxFormUserInput = {
   r032_uplatnujem_na_partnera: true,
 
   /** SECTION Children */
-  r034: [
+  children: [
     {
+      id: 1,
```

Comment:
Treba id? nemoze rodne cislo sluzit ako id?

Reply:
Je tam problem ked pises rodne cislo - re-renderuje sa to pri kazdej zmene cisla (ked napises znak do inputu) a straca sa focus na input.